spec/rails_helper.rb

```
@@ -42,6 +42,9 @@
   # Ensure that if we are running js tests, we are using latest webpack assets
   # This will use the defaults of :js and :server_rendering meta tags
   ReactOnRails::TestHelper.configure_rspec_to_compile_assets(config)
+  
```

RuboCop (1.80.2)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)